### ISSUE TITLE
@change event is triggered when the component loads

### ISSUE THREAD

**Issue by flowser** [vueform/toggle#2]
@change event is triggered when the component loads

**Comment by flowser** [comment]
it cannot be used on list rendering

**Comment by ammardev** [comment]
Same problem. I resolved it by checking if the component is mounted or not:

```html
<toggle
{... other attributes}
@change="myMethod(post.id)"/>
```

```js
data() {
    return {
        mounted: false,
    };
},

mounted() {
    this.mounted = true;
},

methods: {
    updatePublishStatus(id) {
        if (!this.mounted) {
            return;
        }
        // My Ajax code
    }
},
```

**Comment by adamberecz** [comment]
Resolved in `2.0.0`.

**Comment by dieg0** [comment]
This is happening in version 2.0.1, every time a new toggle is loaded, the trueValue is assigned as the model value. It only happens when the model value is assigned from setup, if model is null this doesn't happen (ie only happens on edit form, not in create form)

Had to reset the model value after component mounted
